packages/docs/docs/api/tutorials/bots/logistics-into-ehr.md

```diff
@@ -0,0 +1,153 @@
+
+* Create a [ServiceRequest](https://app.medplum.com/ServiceRequest) when a package is shipped to a [Patient](https://app.medplum.com/Patient)'s home
+* Link a Tracker url (that shows the delivery history of a package) to a ServiceRequest.
+* Receive updates as that shipment is sent to the Patient via [Easypost Webhooks](https://www.easypost.com/docs/api#webhooks) and update the ServiceRequest accordingly.
+* Receive confirmation that the shipment has arrived (also via Easypost Webhooks).
+
+## The Implementation
+
+### Account and Policy Setup
+
+* Make sure you have an account on Medplum, if not, [register](https://app.medplum.com/register).
+* Make sure you have an account on Easypost, if not, [register](https://www.easypost.com/signup)
+* Create a [ClientApplication](https://app.medplum.com/admin/project) on Medplum called "Easypost Webhook".
+* (Optional) Create a very restrictive [AccessPolicy](https://app.medplum.com/AccessPolicy) called "Easypost", make it so that the policy only allows readwrite on the ServiceRequest.
+* (Optional) In the [ProjectAdmin dashboard](https://app.medplum.com/admin/project) apply the "Easypost" policy to the `ClientApplication` by clicking `Access`.
+
+### Bot Setup
+
+* Linking the logistics system, Easypost, to Medplum is done through [Medplum Bots](https://app.medplum.com/Bot).
+* At a high level, webhooks are sent from Easypost to Medplum, updating data in Medplum based on the contents of the Webhook.
+* This example assumes that there is a `ServiceRequest` with an identifier that matches the Easypost `reference`.
+
+* Make the Easypost subscription bot that will listen for webhooks
+  * First, [create a bot](https://app.medplum.com/Bot/new) called EasyPost Shipment Handler and save it
+  * Paste the code below into the Bot you created and save.
+
+  ```js
+  //Grab the reference and the tracker object out of the Easypost Shipment json
+  const reference = input.get('reference');
+  const tracker = input.get('tracker');
+  const status = tracker.get('status');
+  const public_url = tracker.get('public_url');
+
+  //Find the ServiceRequest that has the Easypost reference
+  const serviceRequest = await medplum.readResource('ServiceRequest', reference);
+
+  if (!serviceRequest) {
+    const to_address = input.get('to_address');
+    const shipmentAddress = await medplum.createResource({
+      resourceType: 'Location',
+      identifier: [{
+        system: 'https://www.easypost.com/address',
+        value: to_address.get('id');
+      }],
+      name: to_address.get('name');
+      address: {
+        use: 'home',
+        type: 'physical',
+        line: [to_address.get('street1'), to_address.get('street2')],
+        city: to_address.get('city'),
+        state: to_address.get('state'),
+        postalCode: to_address.get('zip'),
+        country: to_address.get('country')
+      }
+    })
+    const newServiceRequest = await medplum.createResource({
+    resourceType: 'ServiceRequest',
```

Comment:
I'm a bit concerned about making single service as our standard recommendation. It's a bit awkward for this kind of use case (for example, the location only references the `to` , but not the `from`)

I think our general recommendation in public docs should use request/event objects that are closer to the actual use case. For instance, I think we should  use the `SupplyDelivery` resource here

Reply:
The downside of that is that we don't have a nice place to throw the status like `orderDetail`

Reply:
Yeah, the OrderDetail is a concern here.  Also, SupplyDelivery is less mature.  Is the recommendation to move the whole example to SupplyDelivery?

Reply:
Yeah that is the suggestion. Maybe we can fix this in V2 of the tutorial, I think our experience  with some other customers of using one big ServiceRequest on a long-running process with text based statuses has created complexity when in comes to interoperability. Standardizing on the FHIR Event status fields would help automate tracking. 

For human-readable, business-logic statuses, we could use the `status-reason` field. The HL7 group is publishing 
"standard extensions" for the "Event" pattern, and status-reason seems like it will be present on all Event objects (which could help with interop)

http://hl7.org/fhir/event-extensions.html

Reply:
This might be getting ahead of ourselves right now, but I wanted to flag because once these data modeling decisions get baked into a workflow, they can be hard to change.